go.mod

```diff
@@ -89,3 +89,7 @@ replace k8s.io/controller-manager => k8s.io/controller-manager v0.20.0
 replace k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.20.0
 
 replace k8s.io/mount-utils => k8s.io/mount-utils v0.20.0
+
+replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
```

Comment:
This is not how you update dependencies. Let's also use this opportunity to update *all* dependencies to the next minor version, not just those that happen to have known issues.

Please use `go get -t -u ./... && go mod tidy && go mod vendor` and submit that.

Reply:
Thanks for pointing this out, let me re-organize other related PRs as well.
I added in the required version into the required field, then `go get -t -u ./... && go mod tidy && go mod vendor` to upgrade the mod and vendor.